_docs/setup/kubernetes/ansible-install.md

```diff
@@ -1,122 +1,115 @@
-pods, services etc. on to the cluster.
+Operator installs Istio using all defaults on Openshift:
 
-### Note on istio.delete_resources
+```bash
+ansible-playbook main.yml
+```
 
-Activating the `istio.delete_resources` flag will result in any Istio related resources being deleted from the cluster before Istio is reinstalled.
+## Operational overrides
 
-In order to avoid any inconsistency issues, this flag should only be used to reinstall the same version of Istio on a cluster. If a new version
-of Istio need to be reinstalled, then it is advisable to delete the `istio-system` namespace before executing the playbook (in which case the 
-`istio.delete_resources` flag does not need to be activated)  
+There may be circumstances in which defaults require overrides.
 
-## Typical use cases
+The following commands describe how an operator could use overrides with this Ansible playbook:
 
-The following commands are some examples of how a user could install Istio using this Ansible role:
+Operator installs Istio on Kubernetes:
 
-- User executes installs Istio accepting all defaults
 ```bash
-ansible-playbook main.yml
+ansible-playbook main.yml -e '{"cluster_flavour": "k8s"}'
 ```
 
-- User installs Istio on to a Kubernetes cluster 
-```bash
-ansible-playbook main.yml -e '{"cluster_flavour": "k8s"}' 
-```
+Operator installs Istio on Kubernetes and the path to `kubectl` is explicitly set:
 
-- User installs Istio on to a Kubernetes cluster and the path to `kubectl` is expicitly set (perhaps it's not on the PATH)
 ```bash
-ansible-playbook main.yml -e '{"cluster_flavour": "k8s", "cmd_path": "~/kubectl"}' 
+ansible-playbook main.yml -e '{"cluster_flavour": "k8s", "cmd_path": "~/kubectl"}'
 ```
 
-- User wants to install Istio on Openshift with settings other than the default
+Operator installs Istio on Openshift with settings other than the default:
+
 ```bash
 ansible-playbook main.yml -e '{"istio": {"release_tag_name": "0.6.0", "auth": true, "delete_resources": true}}'
 ```
 
-- User wants to install Istio on Openshift but with custom add-on settings
+Operator installs Istio on Openshift with customized addons:
+
 ```bash
 ansible-playbook main.yml -e '{"istio": {"delete_resources": true, "addon": ["grafana", "prometheus", "jaeger"]}}'
 ```
 
-- User wants to install Istio on Openshift and additionally wants to deploy some of the samples
+Operator installs Istio on Openshift and additionally wants to deploy some of the samples:
+
 ```bash
 ansible-playbook main.yml -e '{"istio": {"samples": ["helloworld", "bookinfo"]}}'
 ```
 
-The list of available addons can be found at `istio/vars.main.yml` under the name `istio_all_addons`.
-It should be noted that when Jaeger is enabled, Zipkin is disabled whether or not it's been selected in the addons section.
+Operator installs Jaeger tracing instead of Zipkin tracing:
+
+```bash
+ansible-playbook main.yml -e '{"istio": {"jaeger": true}}'
```

Comment:
This is not correct, but that is totally my fault, since I didn't properly clean up the `jaeger` variable as part of https://github.com/istio/istio/pull/3603.

I will create a new PR to cleanup the code

Reply:
@sdake See https://github.com/istio/istio/pull/4675 which cleans up the unused `istio.jaeger` variable. 

Basically the Jaeger addon is installed in exact same way as the other addons.

Reply:
@geoand as I have not validated these instructions, is your suggestion to remove lines 100->104?

Reply:
Yes, that is exactly what I am suggesting

Reply:
done.